api/descriptions/policies/compliance_container_put.md

```diff
@@ -1,35 +1,49 @@
-Updates all rules that make up your compliance policy for containers in a single shot.
-Updating all rules at the same time makes it possible to maintain strict ordering between rules.
+Updates the compliance policy for containers protected by Defender.
+All rules in the policy are updated in a single shot.
+
+This endpoint maps to the policy table in **Defend > Compliance > Containers and images > Deployed** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+To construct an effective rule for this policy, specify at least one "check" and one `effect` value. 
+See [How to Construct a Compliance Policy](#how-to-construct-a-compliance-policy) for more info.
+
+For a full list of checks, go to **Defend > Compliance > Containers and images > Deployed** in the Console UI and create a new rule.
+All prebuilt checks and their IDs are shown under **Compliance actions**.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/compliance/container' \
+  -k \
+  -X PUT \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -d \
+'{
+   "rules":[
+      {
+      "name": "my-rule",
+      "effect": "alert",
+      "collections":[
+         {
+            "name":"All"
+         }
+      ],
+      "condition": {
+         "vulnerabilities": [
+         		{
+         			"id": 406,
+         			"block": false,
+         			"minSeverity": 1
+         		}
+         	]
+      }
```

Comment:
@KICKdesigns This curl command doesn't work. It's missing the closing brace `}` for the rule object in the `rules[]` array. I'm fixing this for you now. Please follow the QA process I outlined in the other PR.

Reply:
@KICKdesigns Fixed